Mathlib/Algebra/Group/Commute/Units.lean

```diff
@@ -13,10 +13,12 @@ import Mathlib.Algebra.Group.Semiconj.Units
 
 -/
 
-variable {M : Type*} [Monoid M]
+variable {α : Type*}
```

Comment:
Please keep the semantically valuable variable name `M`. If you also need `α` for something that is not a monoid, that's fine. But it's useful to call the monoids in this file something `M`-like.